frontend/src/pages/Landing.jsx

```diff
@@ -106,7 +94,7 @@ export default function Landing() {
           </Row>
           {/*
           <Row>
-            <CustomComponent
+            <OptionButton
               glyph="screenshot"
               header="Neighborhood"
```

Comment:
If the Neighborhood page is removed completely maybe we should remove this too rather than just commenting it out.

Reply:
Removed in 0ba42d1